upgrade/runner.py

```diff
@@ -236,11 +234,16 @@ def product_upgrade(product):
                                 foreman_debug, 'capsule_{}'.format(cap_host),
                                 host=cap_host)
                             raise
+                    # Execute tasks as post upgrade tier1 tests are
+                    # dependent
+                    post_upgrade_test_tasks(sat_host, cap_hosts)
                 if product == 'client' or product == 'longrun':
                     clients6 = setup_dict['clients6']
                     clients7 = setup_dict['clients7']
                     satellite6_client_upgrade('rhel6', clients6)
                     satellite6_client_upgrade('rhel7', clients7)
+                    # Execute tasks as post upgrade tier1 tests are dependent
+                    post_upgrade_test_tasks(sat_host, cap_hosts)
```

Comment:
Same here, please remove from client if loop and keep it only once after all upgrades

Reply:
yes